oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/authentication/BearerTokenAuthenticationFilter.java

```diff
@@ -208,7 +220,7 @@ public void setAuthenticationFailureHandler(final AuthenticationFailureHandler a
 	/**
 	 * Set the {@link AuthenticationDetailsSource} to use. Defaults to
 	 * {@link WebAuthenticationDetailsSource}.
-	 * @param authenticationDetailsSource the {@code AuthenticationConverter} to use
+	 * @param authenticationDetailsSource the {@code AuthenticationDetailsSource} to use
 	 * @since 5.5
 	 */
 	public void setAuthenticationDetailsSource(
```

Comment:
Will you please deprecate this method since `BearerTokenAuthenticationConverter` takes care of this?